content/en/docs/contribute/_index.md

```diff
@@ -4,28 +4,30 @@ title: Contribute to Kubernetes docs
 linktitle: Contribute
 main_menu: true
 weight: 80
+card:
+  name: contribute
+  weight: 10
+  title: Contributing Overview
 ---
 
 {{% capture overview %}}
 
-If you would like to help contribute to the Kubernetes documentation or website,
-we're happy to have your help! Anyone can contribute, whether you're new to the
-project or you've been around a long time, and whether you self-identify as a
-developer, an end user, or someone who just can't stand seeing typos.
+The Kubernetes website loves all contributions from new members and experienced ones! Compared to other parts of the Kubernetes project, documentation is an easy way to
+get involved and have impact, especially for new contributors.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## Getting Started
+## Getting started
 
-Anyone can open an issue describing problems or desired improvements with documentation, or contribute a change with a pull request (PR).
+Anyone can open an issue about documentation, or contribute a change with a pull request (PR).
 Some tasks require more trust and need more access in the Kubernetes organization.
 See [Participating in SIG Docs](/docs/contribute/participating/) for more details about
 of roles and permissions.
 
-Kubernetes documentation resides in a GitHub repository. While we welcome
-contributions from anyone, you do need basic comfort with git and GitHub to
+Kubernetes documentation resides in a GitHub repository.
```

Comment:
[GitHub repository](https://github.com/kubernetes/website)

Reply:
Anyone can open an issue about the Kubernetes documentation, blog, website ...